client/my-sites/stats/site.jsx

```diff
@@ -144,8 +147,30 @@ class StatsSite extends Component {
 		this.props.recordTracksEvent( 'calypso_stats_parsely_banner_click' );
 	};
 
+	buildPrivateSiteBanner() {
+		return (
+			<Banner
+				callToAction={ translate( 'Launch your site' ) }
+				className="stats__private-site-banner"
+				description={ translate(
+					'Changing your site from private to public helps people find you and get more visitors. Don’t worry, you can keep iterating on your site.'
+				) }
+				disableCircle="true"
+				event="calypso_stats_private_site_banner"
+				dismissTemporary="true"
+				dismissPreferenceName="dismiss"
+				href="/settings/general"
```

Comment:
This should specify the current site -- if the user has more than one site, this link drops them into the site selector. You should use the existing `slug` prop for this.
```suggestion
				href={ `/settings/general/${ slug }` }
```